reference/forms/types/birthday.rst

```diff
@@ -1,13 +1,13 @@
 .. index::
    single: Forms; Fields; birthday
 
-birthday Field Type
+Birthday Field Type
```

Comment:
The only issue I see with changing this is that people then may think that the have to use `Birthday` instead of `birthday` in their code.